source/extensions/filters/network/dubbo_proxy/conn_manager.h

```diff
@@ -79,6 +79,9 @@ class ConnectionManager : public Network::ReadFilter,
   void sendLocalReply(MessageMetadata& metadata, const DubboFilters::DirectResponse& response,
                       bool end_stream);
 
+  // This function is for testing only.
+  std::list<ActiveMessagePtr>& getActiveMessagesForTesting() { return active_message_list_; }
```

Comment:
nit: getActiveMessagesForTest (ForTest is the suffix for test only method)

Reply:
done